src/microbe_stage/Biome.cs

```diff
@@ -45,7 +50,7 @@ public class Biome : IRegistryType
 
     public void Check(string name)
     {
-        if (string.IsNullOrEmpty(Name) || string.IsNullOrEmpty(Background))
+        if (string.IsNullOrEmpty(Name) || string.IsNullOrEmpty(Background) || string.IsNullOrEmpty(Panorama))
         {
             throw new InvalidRegistryDataException(name, GetType().Name,
                 "Empty normal or damaged texture");
```

Comment:
Could you fix the error message here? Looks like this is copy-pasted from membrane but someone (probably me, honestly) forgot to update the message.